package_test_run.sh

```diff
@@ -0,0 +1,11 @@
+# This script runs alls tests and examples against the currently
+# installed version of TensorFlow.
+
+set -e
+set -x
+
+# run all test files
+find ./tf_big -name '*_test.py' | xargs -I {} python {}
```

Comment:
Any reason we can't use bazel here?

Reply:
maybe there's a way, but we're testing the final wheel here and not just the individual tests. so you'd have to tell bazel to use the existing wheel instead of compiling it

Reply:
Ahh gotcha, makes sense.